static/js/portico/help.js

```diff
@@ -6,6 +6,8 @@ import {activate_correct_tab} from "./tabbed-instructions";
 function registerCodeSection($codeSection) {
     const $li = $codeSection.find("ul.nav li");
     const $blocks = $codeSection.find(".blocks div");
+    const $skiplink = $codeSection.find(".skip-link");
+    $skiplink.addClass("active"); //Activating the "Skip To Main Content" link
```

Comment:
Please put a space after the `//` in comments.  

Also, can add do a screencast showing the example of this being used for a code block?